pdu/Src/controlStateMachine.c

```diff
@@ -288,36 +125,9 @@ uint32_t criticalFailureWarning(uint32_t event)
 uint32_t criticalFailure(uint32_t event)
 {
     DEBUG_PRINT("Critical Failure %lu: Boards will remain on\n", event);
-    fsmSendEventUrgent(&motorFsmHandle, MTR_EV_Motor_Critical, 10 /* timeout */);
-    fsmSendEventUrgent(&coolingFsmHandle, COOL_EV_Critical, 10 /* timeout */);
-    return MN_STATE_Critical_Failure;
-}
-
-uint32_t lvCuttoff(uint32_t event)
-{
-    DEBUG_PRINT("LV Cuttoff, beginning to shutoff boards\n");
-    // Cooling needs to handle LV Cuttoff differently than HV Critical, as for
-    // critical it can keep cooling on, but for lv cuttoff it needs to turn off
-    fsmSendEventUrgent(&coolingFsmHandle, COOL_EV_LV_Cuttoff, 10 /* timeout */);
-    return startLVCuttoffDelay();
-}
-uint32_t startLVCuttoffDelay()
-{
-    sendDTC_FATAL_LV_CUTTOFF_BOARDS_OFF();
-
-    if (xTimerStart(lvShutdownDelayTimer, 1000) != pdPASS) {
-        ERROR_PRINT("Failed to start lv shutdown delay timer\n");
-        lvShutdown(MN_EV_LV_Cuttoff);
-        return MN_STATE_Boards_Off;
-    }
-    return MN_STATE_LV_Shutting_Down;
-}
-
-
-uint32_t lvShutdown(uint32_t event){
-    DEBUG_PRINT("LV Shutdown: Turning Boards Off\n");
-    turnBoardsOff();
-    return MN_STATE_Boards_Off;
+    // Cooling unaffected
```

Comment:
I think it makes sense to still shutdown cooling if a fatal dtc is received. the BMU disables the DC-DC when a fatal dtc is received (can be seen in line 300 of controlStateMachine.c in the BMU) so we'd probably also want to turn off the cooling.